index.html

```diff
@@ -1,215 +1,330 @@
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "G-PG4ZCTQDV5");
+    </script>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <script
+      src="https://kit.fontawesome.com/5f1b7e3077.js"
+      crossorigin="anonymous"
+    ></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <button type="button" class="nav-1">Hello</button>
+        <button type="button" class="nav-2">Portfolio</button>
+        <button type="button" class="nav-3">About</button>
+        <button type="button" class="nav-4">Contact</button>
+        <a class="top-img"></a>
+        <button type="button" class="menu">
+          <img src="./images/menu.png" alt="menu boton" />
+        </button>
+      </nav>
+    </header>
+    <main>
+      <section class="container">
+        <a class="main-img"></a>
+        <a class="main-img-2"></a>
+        <h1>Jose Herrera</h1>
+        <p>
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look of my works. If you like what you see
+          and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <button type="button" class="bottom">Start collaboration</button>
       </section>
-      <section class="five-project">
-        <img class="lap-images-5" src="./images/laptop_5.png" alt="laptop projecto 5">
-        <img class="lap-images_5" src="./images/laptop_5.png" alt="laptop projecto 5">
-        <div class ="project-5">
-          <h3>Project name goes here</h3>
-          <ul class ="flex">          
-            <li>HTML/CSS</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>          
-          </ul>
-          <button type="button" class ="button-project">See this project ➔</button>
-        </div> 
+      <a class="bottom-img"></a>
+      <section class="main">
+        <h2>Projects</h2>
+        <a class="project-geo-1"></a>
+        <a class="projec-puntos"></a>
+        <a class="project-geo-3"></a>
+        <a class="project-geo-circle"></a>
+        <a class="project-geo-6"></a>
+        <a class="project-geo-7"></a>
+        <!-- change the sections to article tags-->
+        <section class="first-project">
+          <img
+            class="lap-images_1"
+            src="./images/laptop_1.png"
+            alt="laptop-project-1"
```

Comment:
Image text alternatives – added an appropriate description to all project images.